lib/signs_ui_web/controllers/messages_controller.ex

```diff
@@ -136,16 +133,14 @@ defmodule SignsUiWeb.MessagesController do
   end
 
   def play(conn, _params) do
-    with {:ok, visual_zones} <- parse_zones(conn, "visual_zones"),
+    with {:ok, zones} <- parse_zones(conn, "zones"),
          {:ok, visual_data} <- parse_visual_data(conn),
          {:ok, expiration} <- parse_expiration(conn) do
-      [scu_id] = Plug.Conn.get_req_header(conn, "x-scu-id")
-
       %SignsUi.Messages.Audio{
         timestamp: DateTime.utc_now() |> DateTime.to_unix(:millisecond),
         visual_data: visual_data,
-        visual_zones: visual_zones,
-        station: Signs.Config.station_code(scu_id, Enum.at(visual_zones, 0)),
+        zones: MapSet.new(zones, &(String.split(&1, "-") |> List.last())),
+        station: Enum.at(zones, 0) |> String.split("-") |> List.first(),
```

Comment:
It's a small thing but I wonder if we could encode the "knowledge" of this dash-separated format in one place instead of repeating it. Maybe by folding it into `parse_zones`?